easybuild/easyconfigs/p/PCRE/PCRE-8.41-foss-2018a.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'PCRE'
+version = '8.41'
+
+homepage = 'http://www.pcre.org/'
+description = """
+ The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
+ and semantics as Perl 5.
+"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
```

Comment:
@RvDijk Please just remove this easyconfig, we already have `PCRE` `8.41` with `GCCcore/6.4.0`, which is good enough and can serve as dep for SWIG with both `foss/2018a` & `intel/2018a`